### DISCUSSION TITLE
Option to change the color of tabs

### DISCUSSION THREAD

**Discussion by shekhar677** [saadeghi/daisyui#2424] [Ideas / feature requests]
There is no option to change color of tabs

**Comment by saadeghi** [comment]
You can change the colors using CSS variables.  
Here's an example: https://play.tailwindcss.com/Ryc6Lklr8w

Let me know if you have any questions

**Reply by shekhar677** [reply]
Thanks for the quick reply. 👍🏻

**Reply by shekhar677** [reply]
I tried changing color of tabs as suggested but its not working in my case.
here is the example: [https://play.tailwindcss.com/it8QaKOPqx](https://play.tailwindcss.com/it8QaKOPqx)

**Reply by saadeghi** [reply]
`--tab-bg` CSS variable is for `tab-lifted` on [this example](https://play.tailwindcss.com/Ryc6Lklr8w)

for `tabs-boxed` you don't need any of them because it has such a simpler style. Just remove `tab-active` class (because it overrides the primary color. if you don't want primary, you don't need `tab-active`) and simply add a bg color to it
```css
<div class="tabs tabs-boxed">
  <a class="tab">Tab 1</a> 
  <a class="tab bg-yellow-400">Tab 2</a> 
  <a class="tab">Tab 3</a>
</div>
```

**Reply by shekhar677** [reply]
thanks this worked.

**Reply by ZaynTseng** [reply]
Oh thanks a lot for the solution of styling `tabs-boxed` ! Literally dig into everywhere and now the answer is here, simply remove the `tab-active` class and that's it😂
